src/components/Button/index.js

```diff
@@ -109,6 +109,8 @@ const propTypes = {
 
     /** Accessibility label for the component */
     accessibilityLabel: PropTypes.string,
+
+    forwardedRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({current: PropTypes.oneOfType([PropTypes.instanceOf(React.Component), PropTypes.func])})]),
```

Comment:
@iwiznia I see you merged this into you branch – make sure you add a JSDoc for this prop

Reply:
Yep, will do!